packages/editor/src/lib/components/default-components/DefaultTextLabel.tsx

```diff
@@ -0,0 +1,164 @@
+import {
```

Comment:
reviewing more of this diff i noticed a lot of these components were moved deliberately - what was the motivation for that?

Reply:
gotcha, makes sense. yeah i think the boundary is still fuzzy to me so that makes sense the trajectory that we're aiming for w.r.t. to tldraw vs. the editor. thx for the clarfication.

i'll rework to put it in the `tldraw` package. and yeah to your second comment here, i think the only other thing that was moved was `useEditableText` which was to be in the same package as DefaultTextLabel. but, if it's meant to be in `tldraw` then i'll move it back 👍